packages/ckeditor5-list/docs/features/lists.md

```diff
@@ -1,22 +1,20 @@
 ---
 menu-title: Ordered and unordered lists
-meta-title: Ordered and unordered lists | CKEditor 5 Documentation
+meta-title: Lists | CKEditor 5 Documentation
 category: features-lists
 order: 10
-modified_at: 2022-05-12
 ---
 
+{@snippet features/lists-source}
+
 # Ordered and unordered lists
 
-The list feature lets you create ordered (numbered) and unordered (bulleted) lists. You can use ordered lists where the order of the items matters (as in instructions) and unordered lists where it is not that important (as in a list of ingredients).
+The list feature lets you create ordered and unordered lists. The unique thing about them is that you can put any content inside each list item (including block elements like paragraphs and tables), retaining the continuity of numbering and indentation.
 
 <info-box warning>
-	The regular lists feature will be replaced with the new {@link features/document-lists document lists} in the upcoming releases and will be sunset at the beginning of 2024. Consider migrating if you need that functionality. If you wish to switch to the document lists feature, you need to {@link features/document-lists#installation install it} first.
-	See [#14767](https://github.com/ckeditor/ckeditor5/issues/14767) for more details.
+	Since version 40.0.0, the list support plugin has changed for CKEditor&nbsp;5. You can read more about this change in the {@link updating/update-to-40#a-new-default-lists-plugin-coming Update to CKEditor 40.0.0} guide.
 </info-box>
 
-{@snippet features/lists-source}
-
 ## Demo
```

Comment:
Open to discussion: is the demo quite complex. The previous one was basic and short. The new one shows "inside" list capabilities. I wonder if we should simplify it.

Reply:
I think it should show the extended capabilities. This is what we replaced the plugin for.